frontend/src/app/page.tsx

```diff
@@ -16,15 +17,33 @@ export default function Home() {
     getVids();
   }, []);
 
+  useEffect(() => {
+
+    const getProgress = async () => {
+      if (!videos.length) return
+      const inProgressVideoIds = videos.filter(video => video.processing).map(video => video.id).join('&cid=')
+      const resp = await fetch(`/api/clips/progress?cid=` + inProgressVideoIds);
+      const {clips} = (await resp.json()) as Progress
```

Comment:
```suggestion
      const { clips } = (await resp.json()) as Progress
```
You should either lint files while working or before you push. You can also configure your editor to format on every save